templates/conf/fluent.conf.erb

```diff
@@ -82,6 +82,19 @@
   logstash_format "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_FORMAT'] || 'false'}"
   logstash_prefix "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX'] || 'logstash'}"
   logstash_prefix_separator "#{ENV['FLUENT_OPENSEARCH_LOGSTASH_PREFIX_SEPARATOR'] || '-'}"
+  reload_connections "#{ENV['FLUENT_OPENSEARCH_RELOAD_CONNECTIONS'] || 'true'}"
+  reconnect_on_error "#{ENV['FLUENT_OPENSEARCH_RECONNECT_ON_ERROR'] || 'false'}"
+  reload_on_failure "#{ENV['FLUENT_OPENSEARCH_RELOAD_ON_FAILURE'] || 'false'}"
+  log_os_400_reason "#{ENV['FLUENT_OPENSEARCH_LOG_OS_400_REASON'] || 'false'}"
+  target_index_key "#{ENV['FLUENT_OPENSEARCH_TARGET_INDEX_KEY'] || use_nil}"
+  include_timestamp "#{ENV['FLUENT_OPENSEARCH_INCLUDE_TIMESTAMP'] || 'false'}"
+  template_name "#{ENV['FLUENT_OPENSEARCH_TEMPLATE_NAME'] || use_nil}"
+  template_file "#{ENV['FLUENT_OPENSEARCH_TEMPLATE_FILE'] || use_nil}"
+  template_overwrite "#{ENV['FLUENT_OPENSEARCH_TEMPLATE_OVERWRITE'] || use_default}"
+  sniffer_class_name "#{ENV['FLUENT_SNIFFER_CLASS_NAME'] || use_nil}"
+  request_timeout "#{ENV['FLUENT_OPENSEARCH_REQUEST_TIMEOUT'] || '5s'}"
+  application_name "#{ENV['FLUENT_OPENSEARCH_APPLICATION_NAME'] || use_default}"
+  bulk_message_request_threshold "#{ENV['FLUENT_OPENSEARCH_BULK_MESSAGE_REQUEST_THRESHOLD'] || use_nil}"
```

Comment:
The default value of `bulk_message_request_threshold` is `-1`: 
https://github.com/fluent/fluent-plugin-opensearch/blob/6a4b1868808555bb022112becc2746c2e080261f/lib/fluent/plugin/out_opensearch.rb#L95
https://github.com/fluent/fluent-plugin-opensearch/blob/6a4b1868808555bb022112becc2746c2e080261f/lib/fluent/plugin/out_opensearch.rb#L174

It would be better to use `use_default` if unsure the default value.